api/src/v2/Registration/Uploader/OrgDeputyshipUploader.php

```diff
@@ -134,7 +134,7 @@ private function handleOrganisation(OrgDeputyshipDto $dto)
 
             $this->currentOrganisation = $organisation;
 
-            $this->added['organisations'][] = $organisation->getId();
+            $this->added['organisations'][] = $organisation;
```

Comment:
Hmmm, we may need to see what impact this has when running at scale. If we have 1000s of orgs in here we may get out of memory errors.